Assets/Views/Settings.tscn

```diff
@@ -312,10 +312,31 @@ margin_right = 379.0
 margin_bottom = 25.0
 theme = ExtResource( 6 )
 
-[node name="FontSize" type="HBoxContainer" parent="HBX/C/VBoxContainer"]
+[node name="TimeFormatDashboard" type="HBoxContainer" parent="HBX/C/VBoxContainer"]
 margin_top = 308.0
 margin_right = 557.0
-margin_bottom = 335.0
+margin_bottom = 333.0
+
+[node name="Text" type="Label" parent="HBX/C/VBoxContainer/TimeFormatDashboard"]
+margin_top = 4.0
+margin_right = 350.0
+margin_bottom = 21.0
+rect_min_size = Vector2( 350, 0 )
+custom_colors/font_color = Color( 1, 1, 1, 0.611765 )
+custom_fonts/font = ExtResource( 2 )
+custom_styles/normal = SubResource( 22 )
+text = "Set 24h time format"
```

Comment:
I think this could be reworded to be clearer, something like: "24 hour time format", so that it is simpler to read.